### ISSUE TITLE
Add info to Readme about jcenter and current version from bintray 

### ISSUE THREAD

**Issue by avegrv** [RedMadRobot/state-delegator#3]
To get started using `state-delegator` is to add it as a gradle dependency. You need to make sure you have the JCenter repository included in the build.gradle file in the root of your project:

```gradle
repositories {
    jcenter()
}
```

Next add a dependency in the `build.gradle` file of your app module. The
following will add a dependency to the full library:

```gradle
dependencies {
    ...

    implementation "com.redmadrobot:state-delegator:X.Y"
}
```

And that's all! :-)

**Comment by osipxd** [comment]
Shield with version: 
[![Bintray](https://img.shields.io/bintray/v/redmadrobot-opensource/android/state-delegator.svg?style=flat-square)](https://bintray.com/redmadrobot-opensource/android/state-delegator)
(created with https://shields.io/)

**Comment by avegrv** [comment]
or like this
# State Delegator [![Bintray](https://img.shields.io/bintray/v/redmadrobot-opensource/android/state-delegator.svg?style=flat-square)](https://bintray.com/redmadrobot-opensource/android/state-delegator) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)

**Comment by avegrv** [comment]
Fixed at #4
